docs/source/setup/kubernetes.rst

```diff
@@ -0,0 +1,244 @@
+   bald-eel-scheduler-3074430035-cn1dt   1/1       Running             0            1m
+   bald-eel-worker-3032746726-202jt      1/1       Running             0            1m
+   bald-eel-worker-3032746726-b8nqq      1/1       Running             0            1m
+   bald-eel-worker-3032746726-d0chx      0/1       ContainerCreating   0            1m
+
+   $ kubectl get services
+   NAME                 TYPE           CLUSTER-IP      EXTERNAL-IP      PORT(S)                      AGE
+   bald-eel-jupyter     LoadBalancer   10.11.247.201   35.226.183.149   80:30173/TCP                  2m
+   bald-eel-scheduler   LoadBalancer   10.11.245.241   35.202.201.129   8786:31166/TCP,80:31626/TCP   2m
+   kubernetes           ClusterIP      10.11.240.1     <none>           443/TCP
+   48m
+
+You can use the addresses under ``EXTERNAL-IP`` to connect to your now-running
+Jupyter and Dask systems.
+
+Notice the name ``bald-eel``.  This is the name that Helm has given to your
+particular deployment of Dask.  You could, for example, have multiple
+Dask-and-Jupyter clusters running at once and each would be given a different
+name.  You will use this name to refer to your deployment in the future.  You
+can list all active helm deployments with::
+
+   helm list
+
+   NAME            REVISION        UPDATED                         STATUS      CHART           NAMESPACE
+   bald-eel        1               Wed Dec  6 11:19:54 2017        DEPLOYED    dask-0.1.0      default
+
+
+Connect to Dask and Jupyter
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When we ran `kubectl get services` we saw some externally visible IPs
+
+.. code-block:: bash
+
+   mrocklin@pangeo-181919:~$ kubectl get services
+   NAME                 TYPE           CLUSTER-IP      EXTERNAL-IP      PORT(S)                       AGE
+   bald-eel-jupyter     LoadBalancer   10.11.247.201   35.226.183.149   80:30173/TCP                  2m
+   bald-eel-scheduler   LoadBalancer   10.11.245.241   35.202.201.129   8786:31166/TCP,80:31626/TCP   2m
+   kubernetes           ClusterIP      10.11.240.1     <none>           443/TCP                       48m
+
+We can navigate to these from any web browser.  One is the Dask diagnostic
+dashboard.  The other is the Jupyter server.  You can log into the Jupyter
+notebook server with the password, `dask`.
+
+You can create a notebook and create a Dask client from there.  The
+``DASK_SCHEDULER_ADDRESS`` environment variable has been populated with the
+address of the Dask scheduler.  This is available in Python in the ``config`` dictionary.
+
+.. code-block:: python
+
+   >>> from dask.distributed import Client, config
+   >>> config['scheduler-address']
+   'bald-eel-scheduler:8786'
+
+Although you don't need to use this address, the Dask client will find this
+variable automatically.
+
+.. code-block:: python
+   from dask.distributed import Client, config
+   client = Client()
+
+
+Configure Environment
```

Comment:
On thing I don't understand (as a newcomer to kubernetes) is what should the relationship be between nodes and workers? Say I have a cluster with 3 machines, what would a good number of workers be? Is it possible to answer that in general? Are there heuristics I could use to guess?

Reply:
In general you don't think about this.  Kubernetes hides the size of your nodes from you.